### PR TITLE
fix: Fix Chromedriver download for version 115+

### PR DESCRIPTION
**Pull request by joeyparrish**
Chrome has a new distribution system since version 115.

### CODE REVIEW

File: chrome.js [hunk 1]
```diff
@@ -64,10 +65,27 @@ class ChromeWebDriverInstaller extends WebDriverInstallerBase {
...
+      return await InstallerUtils.fetchVersionUrl(
+          `${CDN_URL}/LATEST_RELEASE_${idealMajorVersion}`);
+    } else {
+      const data = await this.getVersionData_();
+
+      // Fall back on the major version if necessary.
+      // We haven't seen this become necessary yet since the new distribution
+      // mechanism debuted, but better safe than sorry.
+      let majorVersion = idealMajorVersion;
+      while (majorVersion >= 115) {
```
Comment (alekzz) [inline chrome.js:80]: I see the number 115 is repeated in the code, better to extract it to a constant.

File: chrome.js [hunk 2]
```diff
@@ -77,12 +95,32 @@ class ChromeWebDriverInstaller extends WebDriverInstallerBase {
    * @return {!Promise}
    */
   async install(driverVersion, outputDirectory) {
+    const majorVersion = parseInt(driverVersion.split('.')[0], 10);
```
Comment (alekzz) [inline chrome.js:98]: nit; `parseInt(driverVersion.split('.')[0], 10)` can be extracted to a function.

File: chrome.js [hunk 3]
```diff
@@ -106,6 +144,73 @@ class ChromeWebDriverInstaller extends WebDriverInstallerBase {
...
   }
+
+  /**
+   * @param {number} majorVersion
+   * @param {string} driverVersion
+   * @param {string} outputDirectory
+   * @return {!Promise}
+   */
+  async installNew_(majorVersion, driverVersion, outputDirectory) {
+    let platform;
```
Comment (alekzz) [inline chrome.js:155]: Line 155-169 can be extracted to a `getPlatform` function

**Review by alekzz — APPROVED** [review]

File: chrome.js (hunk 3 above)
Comment (joeyparrish) [reply chrome.js:155]: No, the platform strings used in the old and new styles differ slightly.

File: chrome.js (hunk 2 above)
Comment (joeyparrish) [reply chrome.js:98]: parseInt is already a function, right?

File: chrome.js (hunk 1 above)
Comment (joeyparrish) [reply chrome.js:80]: Done

File: chrome.js (hunk 3 above)
Comment (joeyparrish) [reply chrome.js:155]: Ex: Before 115, it was mac_arm64 for Arm and mac64 Intel.  From 115 on, it's mac-arm64 and mac-x64.
Ex: Before 115, it was win32 for Windows.  From 115 on, it's win64.

**Pull request merged**